lib/publiccloud/instance.pm

```diff
@@ -602,6 +602,27 @@ sub network_speed_test() {
     record_info("curl $rmt_host", $self->run_ssh_command(cmd => "curl -w '$write_out' -o /dev/null -v https://$rmt_host/", proceed_on_failure => 1));
 }
 
+sub cleanup_cloudinit() {
+    my ($self) = @_;
+    $self->ssh_assert_script_run('sudo cloud-init clean --logs');
+    if (check_var('PUBLIC_CLOUD_CLOUD_INIT_FILE', 'cloud-init.yaml')) {
+        $self->ssh_assert_script_run('sudo rm /root/test_cloud-init.txt');
```

Comment:
Let's use a variable for `/root/test_cloud-init.txt`

Reply:
Really? The `/root/test_cloud-init.txt` is used only once in this subroutine.

Reply:
True. Forget about it.